### ISSUE TITLE
Tibetan Monasteries Accessibility Audit

### ISSUE THREAD

**Issue by erinchambers** [wkdewey/tibetan-monasteries#55]
**Any existing accessibility issues for Tibetan Monasteries will be coming directly from Orchid, so this issue is mainly just a placeholder for representing TM work on the CDRH ADA Title II Project roadmap.**

---

# Version of site audited

**URL:** 
**branch:** ``

# Checklist

  - [ ] Generate lists for this issue using the [Checklist of Products for ADA Title II](https://uofnebraska.sharepoint.com/:x:/r/sites/UNL-UniversityLibraries/DISC/cdrh/Shared%20Documents/CDRH%20Projects/Project%20Folders/Accessibility%20Audits/2025-2026%20ADA%20Title%20II%20Project/Planning%20Documents/Checklist%20of%20Products%20for%20ADA%20Title%20II%20Project.xlsx?d=w52aafefaa1a841b09bae916225c22cbb&csf=1&web=1&e=mvYOdi) spreadsheet
  - [ ] Identify sections and URLs that duplicate search/browse for each section
  - [ ] Identify representative items/pages from each section
  - [ ] Identify any other unique, non-item pages within each section
  - [ ] Compile in spreadsheet (linked above)

## `robots.txt`
  - [ ] Upload `robots.txt`. Include:
    - for each section: search/browse/other views with paginated results

### Contents of `robots.txt` for this site:

```
User-agent: *
Disallow: /faceted-browse/
```

## Site crawl

- [ ] Scan site (dev site if possible) using [WAE tool](https://wae.unl.edu/)

```

```

## Page scans

Pages scanned individually using axeDevTools web browser extension:

## Manual testing

Pages tested manually for things that automated scans cannot detect:

  - [ ] Home page
  - [ ] Primary "static" pages
    - [ ] 

## Delete robots.txt

- [ ] Remove robots.txt from server

**Comment by erinchambers** [comment]
Complete
